spec-source/LemurUtils.lua

```diff
@@ -0,0 +1,26 @@
+local LemurUtils = {}
+
+function LemurUtils.LoadRobase(habitat)
+	local ReplicatedStorage = habitat.game:GetService("ReplicatedStorage")
+
+	-- The paths are relative to the current working directory, which is "robase".
+	local MODULES_TO_LOAD = {}
```

Comment:
At the moment, there is only one proper test and it is not using any other modules. Thought I'd leave this in, in case our tests require some other util modules.